c7n/resources/iam.py

```diff
@@ -257,6 +257,7 @@ class resource_type(TypeInfo):
         # Denotes this resource type exists across regions
         global_resource = True
         arn = 'Arn'
+        config_id = 'UserId'
```

Comment:
Nice, thanks for including this!

(I've got a couple related changes in #7530 that might need to be split out into a separate PR :thinking: ...)